.github/workflows/main.yml

```diff
@@ -165,72 +137,10 @@ jobs:
         if: ${{ success() }}
         run: bash <(curl -s https://codecov.io/bash)
 
-  cleanup:
-    name: Cleanup stale CI namespaces
-    runs-on: ubuntu-latest
-    needs: build
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v1
-      - name: Restore Module Cache
-        uses: actions/cache@v2
-        with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-gomod2-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-gomod2-
-      - name: Restore Build Cache
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/go-build
-          key: ${{ runner.os }}-gobuild-${{ hashFiles('**/*.go') }}
-      - name: Setup Go 1.14
-        uses: actions/setup-go@v1
-        with:
-          go-version: 1.14
-      - name: Cleanup stale CI namespaces
-        run: |
-          touch .env  # it is OK for this file to be empty - needed by Makefile
-          mkdir -p ".kube"
-          echo "$KUBE_CONFIG" | base64 -d > "$KUBECONFIG"
-          go run ./ci/cmd/cleanup/cleanup-namespaces.go
```

Comment:
@nojnhuh it would be nice to resurrect this to ensure that we remove stale namespaces from the CI cluster (unless we have implemented a different mechanism of doing the same)

Reply:
@draychev Each integration test is run on a freshly spun up kind cluster, so there won't ever be old namespaces since the clusters only live as long as the CI job.